addons/kube-proxy/serviceaccount.yaml

```diff
@@ -12,8 +12,10 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
+{{ if ne .Cluster.Network.ProxyMode "ebpf" }}
```

Comment:
I just hope someone from the kube-proxy team doesn't implement ebpf backend for it too.. :)

Reply:
then we change it to the `cni` proxy mode :)

Reply:
.. or add CNI type into the condition